python-package/rgf/fastrgf_model.py

```diff
@@ -569,6 +467,34 @@ def __init__(self,
         self._fitted = None
         self._target = "BINARY"
 
+    _classifier_specific_values = {
+        '{%estimator_type%}': 'classifier',
+        '{%loss_parameter%}': """
+loss : string ("LS" or "MODLS" or "LOGISTIC"), optional (default="LS")
+    Loss function.
+    LS: Least squares loss.
+    MODLS: Modified least squares loss.
+    LOGISTIC: Logistic loss.
+    (Original name: dtree.loss.)
+""",
+        '{%calc_prob_parameter%}': """
+calc_prob : string ("sigmoid" or "softmax"), optional (default="sigmoid")
+    Method of probability calculation.
+""",
+        '{%estimators_property_type_desc%}': 'list of binary classifiers',
+        '{%classes_property%}': """
+classes_ : array of shape = [n_classes]
+    The classes labels when `fit` is performed.
+""",
+        '{%n_classes_property%}': """
+n_classes_ : int
+    The number of classes when `fit` is performed.
+"""
+    }
+    __doc__ = fastrgf_estimator_docstring_template
+    for _template, _value in _classifier_specific_values.items():
```

Comment:
why these variable start from `_`?

Reply:
Due to encapsulation. These variables a private and shouldn't be accessed by users.

Reply:
I see.
_template is accessable because of its scope.